client.lua

```diff
@@ -1353,7 +1415,43 @@ Citizen.CreateThread(function()
 						end
 					end
 
-					
+					-- siren park kill
+					if UseParkKill(veh) then
+						if IsControlPressed(0, 75) then
+							SetLxSirenStateForVeh(veh, newstate)
```

Comment:
Here's an alternative if you'll have issues with siren sync:
```lua
Citizen.CreateThread(function()
	while true do
		Citizen.Wait(0)
		local ped = GetPlayerPed(-1)
		local veh = GetVehiclePedIsIn(ped, true)
		--if  player_is_emerg_driver then
			--if  IsPedSittingInVehicle(ped, veh) and GetPedInVehicleSeat(veh, -1) == PlayerPedId() and ParkKill(veh) and GetIsTaskActive(ped, 2) and GetVehiclePedIsIn(ped, true) then
			if  GetPedInVehicleSeat(veh, -1) == ped and IsPedInAnyVehicle(ped, false) and ParkKill(veh) and GetIsTaskActive(ped, 2) and state_lxsiren[veh] ~= 0 then
				print("po")
				TriggerServerEvent('InteractSound_SV:PlayOnSource', 'shifterleo', 0.3)
				SetLxSirenStateForVeh(veh, 0)
				SetAirManuStateForVeh(veh, 0)
				
				count_bcast_timer = delay_bcast_timer
				Citizen.Wait(1000)
			--end
		end
	end
end)
```

I assume that it's a wacky code but it does the job, if someone would like to clean it up - you're more than welcome to do so.